Statut HSTrójmiasto.tex

```diff
@@ -0,0 +1,254 @@
+	    \item Do kompetencji Komisji Rewizyjnej należy:
+	      \begin{enumerate}
+	        \item kontrolowanie działalności Zarządu,
+	        \item składanie sprawozdań z kontroli na Walnym Zebraniu Członków,
+	        \item składanie wniosków o udzielenie absolutorium dla Zarządu Stowarzyszenia,
+	        \item składanie sprawozdań ze swojej działalności na Walnym Zebraniu Członków,
+	        \item zatwierdzanie rocznego sprawozdania finansowego.
+	      \end{enumerate}
+	    \item Członek Komisji Rewizyjnej może zrezygnować z pełnienia funkcji poprzez złożenie oświadczenia na Walnym Zebraniu Członków.
+	\end{enumerate}
+\section{MAJĄTEK I FUNDUSZE}
+  \begin{enumerate}
+    \item Majątek Stowarzyszenia składa się z:
+      \begin{enumerate}
+        \item składek członkowskich,
+        \item subwencji, darowizn, spadków i zapisów,
+        \item wpływów z odpłatnej działalności statutowej,
+        \item wpływów z ofiarności publicznej,
+        \item dochodów z majątku, odsetek oraz kapitału,
+        \item dotacji,
+        \item wpływów z loterii, aukcji i sponsoringu,
+        \item zbiórek publicznych.
+      \end{enumerate}
+    \item Stowarzyszenie prowadzi gospodarkę finansową zgodnie z obowiązującymi przepisami.
+    \item Organem kompetentnym w zakresie zarządzania majątkiem Stowarzyszenia jest Zarząd.
+    \item Zarząd zobowiązany jest dołożyć wszelkich starań w celu utrzymania zapasu środków na kontach Stowarzyszenia wystarczającego na pokrycie stałych zobowiązań Stowarzyszenia przez okres co najmniej dwóch miesięcy.
+    \item Do zawierania umów, zwłaszcza w sprawach majątkowych i niemajątkowych, udzielania pełnomocnictwa i składania innych oświadczeń woli, w szczególności w sprawach majątkowych upoważnieni są:
+      \begin{enumerate}
+        \item dwóch członków Zarządu działających łącznie,
+        \item każdy członek zarządu działający samodzielnie, jeżeli wysokość podejmowanego zobowiązania nie przekracza 64 PLN\@
+      \end{enumerate}
+    \item Stowarzyszenie nie prowadzi działalności gospodarczej.
+    \item Składki członkowskie ustanawiane są uchwałą Walnego Zebrania Członków.
+  \end{enumerate}
+\section{POSTANOWIENIA KOŃCOWE}
+  \begin{enumerate}
+    \item Uchwałę o rozwiązaniu Stowarzyszenia podejmuje Walne Zebranie Członków większością 2/3 głosów, przy obecności co najmniej połowy członków uprawnionych do głosowania.
+    \item W przypadku podjęcia uchwały o rozwiązaniu Stowarzyszenia przed upływem trzech lat od jego rejestracji majątek stowarzyszenia zostanie przekazany na rzecz Fundacji Code:me zarejestrowanej w Krajowym Rejestrze Sądowym pod numerem 0000613580.
+    \item W przypadku podjęcia uchwały o rozwiązaniu Stowarzyszenia po upływie trzech lat od jego rejestracji Walne Zebranie Członków określa sposób jego likwidacji oraz przeznaczenie majątku Stowarzyszenia.
+    \item W sprawach nie uregulowanych w niniejszym statucie zastosowanie mają przepisy ustawy Prawo o Stowarzyszeniach.
```

Comment:
Czy ten punkt nie jest redundantny? A jeżeli ktoś się upiera, że nie jest to możemy dopisać kodeks cywilny i konstytucję RP? :D

Reply:
Co jest redundatne dokladnie? Jak nie masz spisanego procesu rozwiazania stow to musisz zrobic wpierw walne by zmienic statut, zlozyc wniosek do KRS, poczekac po czym znow miec walne by rozwiazac stow. Dlatego masa stowarzyszen zombie jest bo tego nie wpisali do statutu.